hyperion/app/creation_steps/step8_review_and_refine/consistency_check/models.py

```
@@ -0,0 +1,172 @@
+        "TEST_DATA_INCONSISTENT": ConsistencyIssueCategory.TEST_DATA_INCONSISTENT,
+        "GRADING_CRITERIA_CONFLICT": ConsistencyIssueCategory.GRADING_CRITERIA_CONFLICT,
+        "TEST_METHOD_NAMING_CONFLICT": ConsistencyIssueCategory.TEST_METHOD_NAMING_CONFLICT,
+        "COGNITIVE_LEVEL_MISMATCH": ConsistencyIssueCategory.COGNITIVE_LEVEL_MISMATCH,
+        "SCAFFOLDING_DISCONTINUITY": ConsistencyIssueCategory.SCAFFOLDING_DISCONTINUITY,
+        "PREREQUISITE_ASSUMPTION_VIOLATION": ConsistencyIssueCategory.PREREQUISITE_ASSUMPTION_VIOLATION,
+        "LEARNING_OBJECTIVE_CONTRADICTION": ConsistencyIssueCategory.LEARNING_OBJECTIVE_CONTRADICTION,
+        "COMPLEXITY_PROGRESSION_VIOLATION": ConsistencyIssueCategory.COMPLEXITY_PROGRESSION_VIOLATION,
+        "SKILL_TRANSFER_IMPEDIMENT": ConsistencyIssueCategory.SKILL_TRANSFER_IMPEDIMENT
+    }
+
+    type_map = {
```

Ruff (0.11.9)

Local variable `type_map` is assigned to but never used

Remove assignment to unused variable `type_map`

(F841)